### PR TITLE
[CV Eval] Test for CV Eval/BZ

### PR DESCRIPTION
**Pull request by sambible**
### Problem Statement
Test Coverage for https://bugzilla.redhat.com/show_bug.cgi?id=2137318/https://issues.redhat.com/browse/SAT-15185

### Solution
Added Test + support for `content view version purge`

trigger: test-robottelo
pytest: tests/foreman/cli/test_contentview.py -k 'test_cv_version_purge'

### CODE REVIEW

**Comment by sambible** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_contentview.py -k 'test_cv_version_purge'

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 7545
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/cli/test_contentview.py -k test_cv_version_purge --external-logging
Test Result : ========= 1 passed, 116 deselected, 125 warnings in 808.76s (0:13:28) ==========
```

File: robottelo/cli/contentview.py [hunk 1]
```diff
@@ -113,6 +114,12 @@ def publish(cls, options, timeout=1500000):
         cls.command_sub = 'publish'
         return cls.execute(cls._construct_command(options), ignore_stderr=True, timeout=timeout)
 
+    @classmethod
+    def purge(cls, options, timeout=1500000):
```
Comment (vsedmik) [inline robottelo/cli/contentview.py:118]:
```suggestion
    def purge(cls, options, timeout='25m'):
```
Would be nicer (in whole module).

File: tests/foreman/cli/test_contentview.py [hunk 2]
```diff
@@ -3208,6 +3208,42 @@ def test_show_all_repo_ids(self, module_org, module_product, module_target_sat):
...
+        per page value
+
+        :Verifies: SAT-15185
+
+        :customerscenario: true
+        """
+        content_view = module_target_sat.cli_factory.make_content_view(
+            {'organization-id': module_org.id}
+        )
+        for _ in range(50):
```
Comment (vsedmik) [inline tests/foreman/cli/test_contentview.py:3233]:
```suggestion
        cv_count = 50
        for _ in range(cv_count):
```

File: tests/foreman/cli/test_contentview.py [hunk 3]
```diff
@@ -3208,6 +3208,42 @@ def test_show_all_repo_ids(self, module_org, module_product, module_target_sat):
...
+            module_target_sat.cli.ContentView.publish({'id': content_view['id']})
+        cvv = module_target_sat.cli.ContentView.version_list(
+            {'content-view-id': content_view['id']}
+        )
+        assert len(cvv) == 50
```
Comment (vsedmik) [inline tests/foreman/cli/test_contentview.py:3238]:
```suggestion
        assert len(cvv) == cv_count
```

File: tests/foreman/cli/test_contentview.py [hunk 4]
```diff
@@ -3208,6 +3208,42 @@ def test_show_all_repo_ids(self, module_org, module_product, module_target_sat):
...
+        response = module_target_sat.cli.ContentView.purge({'id': content_view['id']})
+        assert response
```
Comment (vsedmik) [inline tests/foreman/cli/test_contentview.py:3240]:
```suggestion
        assert all(
            [
                f"Version '{v+1}.0' of content view '{content_view.name}' deleted." in response
                for v in range(cv_count - 4)
            ]
        )
```

**Review by vsedmik — COMMENTED** [review]
Just a few proposals for improvement, otherwise looks good to me.

**Comment by sambible** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_contentview.py -k 'test_cv_version_purge'

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 7623
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/cli/test_contentview.py -k test_cv_version_purge --external-logging
Test Result : ========= 1 passed, 117 deselected, 126 warnings in 949.80s (0:15:49) ==========
```

**Review by damoore044 — APPROVED** [review]
ack lgtm

**Review by vsedmik — APPROVED** [review]

**Review by Gauravtalreja1 — APPROVED** [review]

**Pull request merged**